lib/src/models/Cell.dart

```diff
@@ -23,13 +29,42 @@ class Cell extends ICell {
 
   /// Constructs new Cell at [position] with optional [_value]
   /// If value is provided, then cell is flagged as prefilled
-  Cell(position, [this._value = 0]) : super(position) {
+  Cell(this.position, [this._value = 0]){
     _isPrefill = (_value != 0);
     _isValid = _isPrefill;
 
     _onChange = new StreamController.broadcast();
   }
 
+  Cell._({int value,
+    bool isPrefill,
+    bool isValid,
+    bool isMarkup,
+    bool isPristine,
+    this.position}) : this._isPrefill = isPrefill, this._value = value, this._isValid = isValid, this._isMarkup = isMarkup, this.isPristine = isPristine {
+    _onChange = new StreamController.broadcast();
+  }
```

Comment:
```suggestion
  Cell._({int value,
    bool isPrefill,
    bool isValid,
    bool isMarkup,
    bool isPristine,
    Position position}) : this._isPrefill = isPrefill, this._value = value, this._isValid = isValid, this._isMarkup = isMarkup, super(position, isPristine) {
    _onChange = new StreamController.broadcast();
  }
```

_Part of the 'let's keep ICell' movement ;)_